include/boost/math/interpolators/detail/bilinear_uniform_detail.hpp

```diff
@@ -42,7 +45,7 @@ class bilinear_uniform_imp
         y0_ = y0;
         dx_ = dx;
         dy_ = dy;
-        using std::to_string;
+
```

Comment:
Wait, if we don't have the `using std::to_string`, then we won't get the ADL for multiprecision types on the error message on line 50 correct?

See https://github.com/boostorg/multiprecision/pull/339